cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,123 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticleEditorPageObject from '../support/pages/articleEditor.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
 
-  });
+const btnNames = require(`../fixtures/buttonNames.json`);
+const articleTip = require(`../fixtures/articleTip.json`);
+
+const articleEditor = new ArticleEditorPageObject();
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
 
+describe('Article page', () => {
   beforeEach(() => {
+    cy.wrap(Cypress.currentTest.title).as('testTitle');
+
     cy.task('db:clear');
+
+    cy.task('generateUserData').as('userData').then((user) => {
+      cy.authorization(user).then(({ id }) => {
+        cy.task('generateArticle').as('articleData').then((article) => {
+          cy.createArticle(article, id).then(({ slug }) => {
+            new ArticlePageObject(slug).visit();
+
+            cy.wrap(slug)
+              .as('articleSlug');
+          });
+        });
+      });
+    });
+
+    cy.task('generateArticle').as('newArticleData');
   });
 
-  it('should be created using New Article form', () => {
+  it(`should allow to edit an article using Edit button`, function () {
```

Comment:
I have tried to follow the Cypress Best Practices to use the aliases instead of '_**const, let or var**_'. And usage of regular function syntax (_function () {}_) helps to access aliased data using **_this_**. But in some tests an arrow function is used